rl_studio/agents/__init__.py

```diff
@@ -1,20 +1,28 @@
 from rl_studio.agents.agents_type import AgentsType
+from rl_studio.agents.algorithms_type import AlgorithmType
 from rl_studio.agents.exceptions import NoValidTrainingType
 
 
 class TrainerFactory:
     def __new__(cls, config):
 
         agent = config.agent["name"]
+        algorithm = config.algorithm["name"]
 
         if agent == AgentsType.F1.value:
-            from rl_studio.agents.f1.train_qlearn import F1Trainer
-            return F1Trainer(config)
+
+            if algorithm == AlgorithmType.QLEARN.value:
+                from rl_studio.agents.f1.train_qlearn import F1Trainer
+                return F1Trainer(config)
+                
+            elif algorithm == AlgorithmType.DDPG.value:
+                from rl_studio.agents.f1.train_ddpg import F1TrainerDDPG
+                return F1TrainerDDPG(config)    
```

Comment:
We should move this code to the Algorithm factory. You can move it to [this file](https://github.com/JdeRobot/RL-Studio/blob/main/rl_studio/algorithms/__init__.py). Thus, we have separate the "Algoritm" and the "Agents".

How do you see it?